taskd_services/http/api.py

```diff
@@ -33,6 +33,7 @@ def default(self, obj):
 app = Flask(__name__)
 app.config["SQLALCHEMY_DATABASE_URI"] = "sqlite:///" + CERT_DB_PATH
 app.config["RESTFUL_JSON"] = {"cls": TaskdJsonEncoder}
+app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = True
```

Comment:
I'm a little surprised about this change here -- although I don't really care about performance in this particular part of the system at all, I wonder if there was there something you ran into that made this necessary?  I hadn't realized we might be using the event system.